app/routes/_mainLayout.t8_.$character.nav.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavLink } from '@remix-run/react'
+export interface ObjectParams {
+  link: String
+  value: String
+}
+interface Props {
+  navData: ObjectParams[]
+}
+
+function Nav({ navData }: Props) {
+  return (
+    <nav className="flex gap-3">
+      {navData.map((obj, index) => (
+        <NavLink key={index} to={obj.link.toString()}>
```

Comment:
dont think you need to call to string here since link is already a string?
```suggestion
        <NavLink key={index} to={obj.link}>
```

Reply:
Thank you for the suggestions! I have made all the necessary changes:
1. I have removed comments from the file.
2. I moved the file from "app/routes" to "app/components" and renamed it to "Nav.tsx."
3. Actually, I had used 'String' in the interface instead of 'string,' so the 'to' was giving a type error because it accepts a string. And now I have also changed that to string.